govc/tags/category/create.go

```diff
@@ -57,12 +57,12 @@ func (cmd *create) Usage() string {
 }
 
 func (cmd *create) Description() string {
-	return ` Create tag category. 
+	return `Create tag category.
 
 This command will output the ID you just created.
 
 Examples:
-  govc tags.category.create -d "description" -t "categoryType" -m(if set, the cardinality will be true) NAME`
+  govc tags.category.create -d "description" -t "categoryType" NAME`
```

Comment:
We can keep this -m flag and just remove the brackets?
`govc tags.category.create -d "description" -t "categoryType" -m NAME `

Reply:
Yes, done.